<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-2</y>
      <width>1151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(16, 91, 151, 255), stop:1 rgba(44, 156, 234, 255));</string>
    </property>
    <property name="text">
     <string>     Simple Attendance Record System</string>
    </property>
    <property name="margin">
     <number>15</number>
    </property>
   </widget>
   <widget class="QStackedWidget" name="mainWidget">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>70</y>
      <width>1171</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="StudentLoginPage">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <widget class="QWidget" name="widget_3" native="true">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>80</y>
           <width>841</width>
           <height>321</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;</string>
         </property>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>221</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Attendance Login</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Student ID:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="studID">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>110</y>
            <width>801</width>
            <height>31</height>
           </rect>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly</set>
          </property>
         </widget>
         <widget class="QLabel" name="studentValid">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>160</y>
            <width>801</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: red;</string>
          </property>
          <property name="text">
           <string>Validations</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="studentLoginButton">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>240</y>
            <width>801</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: lightblue;</string>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
         <widget class="QLabel" name="beenLogin">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>10</y>
            <width>191</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>You have been login:</string>
          </property>
         </widget>
         <widget class="QLabel" name="nameTime">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>70</y>
            <width>191</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="adminLoginButton">
         <property name="geometry">
          <rect>
           <x>1070</x>
           <y>590</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Login Admin</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="AdminLoginPage">
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>70</y>
        <width>331</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">b</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>331</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: lightblue;</string>
       </property>
       <property name="text">
        <string>Admin Login</string>
       </property>
       <property name="margin">
        <number>20</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="usernameln">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>291</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordln">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>291</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="mainLogin">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>291</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QLabel" name="adminValid">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>291</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: red;</string>
       </property>
       <property name="text">
        <string>Validation in admin</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="mainPage">
     <widget class="QPushButton" name="homeButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QPushButton" name="recordButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Record</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="currentText">
       <string>Accounts</string>
      </property>
      <item>
       <property name="text">
        <string>Accounts</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Admin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Student</string>
       </property>
      </item>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>1111</width>
        <height>551</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="homePage">
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:lightblue;</string>
        </property>
        <property name="text">
         <string>    Home</string>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>1051</width>
          <height>481</height>
         </rect>
        </property>
        <property name="text">
         <string>Insert image here</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="accountsPage">
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:lightblue;</string>
        </property>
        <property name="text">
         <string>    Accounts/[Depends on combo]</string>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>1091</width>
          <height>491</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="adminAccounts">
         <widget class="QPushButton" name="addNewAdmin">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Add New</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="studentAccounts">
         <widget class="QPushButton" name="addNewStudent">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Add New</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="recordPage">
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:lightblue;</string>
        </property>
        <property name="text">
         <string>    Record</string>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="logoutButton">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>10</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
